CRAMv3.tex

```diff
@@ -920,143 +922,363 @@ \section{\textbf{Record structure}}
 other hand, data series that do not require binary coding can be stored separately 
 in external blocks with some other compression applied to them independently.
 
+As CRAM data series may be interleaved within the same blocks\footnote{Interleaving can sometimes provide better compression, however it also adds dependency between types of data meaning it is not possible to selectively decode one data series if it co-locates with another data series in the same block.} understanding the order in which CRAM data series must be decoded is vital.
```

Comment:
No, we can use the same external block ID and interleave to external blocks too.  This can improve compression ratios too if we are putting data with similar frequency distributions in the same block or data with strong correlation between values.

Eg imagine we had one tag X0 which is number of mismatches in this read and X1 which is number of mismatches in this plus mate.  There will be a strong correlation between the value of X1 and the value of X0 (at the very least X1 >= X0), so putting both together would mean an order-1 entropy encoder will probably do a decent job.

In practice we don't do this any more, so it remains untapped potential.

Reply:
Interesting!  I didn't realize that, thanks.